tests/unit/testUncaught.js

```diff
@@ -50,6 +52,13 @@ test('Uncaught handler setup', function (t) {
 });
 
 test('Test uncaught shutdown behavior', function (t) {
+  if (!isString(process.env.GOOGLE_APPLICATION_CREDENTIALS)
```

Comment:
It may be better to fail if the variables aren't set. If this is being run automatically on ci, this test could start skipping without being detected unless the output is inspected manually (unless maybe travis reports skipped tests in some way)?